### ISSUE TITLE
[docs] Property versus Items

### ISSUE THREAD

**Issue by floggle** [zircote/swagger-php#627]
I've spent the last few days trying to get my head around swagger-php and I feel I'm nearly there but the lack of docs is really confusing.
In particular, it's unclear what the difference between OA\Items and OA\Properties is. They seem to be used interchangably.
This file for example has a schema that has a property and a schema that has an items:
https://github.com/zircote/swagger-php/blob/26c9e75fdec48aa07ed7d6dc255ad2fa10250ece/Examples/swagger-spec/petstore/api.php

And on top of those two, there's a Schema defined here that doesn't use either property or items but just plain defines something inside it (the TestProperty schema):
https://github.com/zircote/swagger-php/blob/5d96b7c4155549e32d2715807d0e8cc61020ab2a/tests/Fixtures/DynamicReference.php

In fact, it's so bad I've had to write my own "docs" to try and understand what the endpoints are.
If you want my community contribution toward the docs, this is it (and I'm fairly sure it's wrong, and certain it's incomplete):

------------

@OA\Get
	Is request for a given path
	Can include (or ref) @OA\Parameter - which lists the allowed parameters
	Can include (or ref) @OA\Response - which lists the various response codes and what they equate to

@OA\Parameter
	A request parameter. i.e. Path, query, cookie, or header.
	Can include (or ref) @OA\Schema - which specifies a schema

@OA\Schema
	Defines a Schema (structure) for something
	Can include @OA\Items - which is the cover the actual items
	Can include (or Ref) other @OA\Schema - this allows for both nested schemas, and merging schemas together
	Can use "allOf" to include many other schemas - https://github.com/zircote/swagger-php/blob/26c9e75fdec48aa07ed7d6dc255ad2fa10250ece/Examples/swagger-spec/petstore-with-external-docs/models/Pet.php

@OA\Property
	Goes inside a @OA\schema
	Defines a response field.
	???

@OA\Items
	Define an item in a schema
	Can ref a @OA\Schema

@OA\Response
	Specifies what to expect from the response
	Can include @OA\JsonContent

@OA\JsonContent
	Same as schema, but for the output which it's defining as being JSON (under a "application/json" object key)
	As @OA\Schema
	Basically just ref this to a schema

**Comment by bfanger** [comment]
`@OA\Items` are used when the `@OA\Schema`, `@OA\Property` or other Schema-like annotations  are of `type="array"`. The `@OA\Items` describe the elements inside the array.

Thanks for the documentation snippets,  I see what I can salvage for an annotations chapter.
